packages/syft/src/syft/ast/klass.py

```diff
@@ -532,22 +615,6 @@ def create_pointer_class(self) -> None:
         setattr(klass_pointer, "path_and_name", self.path_and_name)
         setattr(self, self.pointer_name, klass_pointer)
 
-    def store_init_args(outer_self: Any) -> None:
```

Comment:
@madhavajay this is ok to be removed?

Reply:
Looks like its being called in transformers, so im adding it back.